flexibleengine/resource_flexibleengine_dli_queues_v1_test.go

```diff
@@ -0,0 +1,92 @@
+package flexibleengine
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/huaweicloud/golangsdk/openstack/dli/v1/queues"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/huaweicloud/golangsdk"
+)
+
+func TestAccDliQueueV1_basic(t *testing.T) {
```

Comment:
It is suggested to adding more test cases , such as modification , to check more attributes .